Core/include/Acts/Vertexing/ImpactPointEstimator.hpp

```diff
@@ -21,20 +21,27 @@
 namespace Acts {
 
 struct ImpactParametersAndSigma {
-  double IPd0 = 0.;
-  double IPz0 = 0.;
-  double IPz0SinTheta = 0.;
-  double sigmad0 = 0.;
-  double sigmaz0 = 0.;
-  double sigmaz0SinTheta = 0.;
-  double PVsigmad0 = 0.;
-  double PVsigmaz0 = 0.;
-  double PVsigmaz0SinTheta = 0.;
+  // Impact parameters ...
+  double d0 = 0.;
+  double z0 = 0.;
+  // ... and their standard deviations wrt a vertex, e.g.:
+  // sigmaD0 = sqrt(Var(X) + Var(Y) + Var(d0)),
+  // where X and Y are the x- and y-coordinate of the vertex
+  double sigmaD0 = 0.;
+  double sigmaZ0 = 0.;
+  // Absolute difference in time between the vertex and the track at the 2D PCA
+  // ...
+  std::optional<double> deltaT = std::nullopt;
+  // ... and standard deviation wrt a vertex
+  std::optional<double> sigmaDeltaT = std::nullopt;
 };
 
 /// @class ImpactPointEstimator
 ///
 /// @brief Estimator for impact point calculations
+/// A description of the underlying mathematics can be found here:
+/// https://github.com/acts-project/acts/pull/2414
+/// TODO: Upload reference at a better place
```

Comment:
I just discussed with @paulgessinger and there's no hard rule concerning TODO's on the main branch. In this case I would tend to leave it because uploading a reference with the PR is not a good practice and I need to find a better solution...

Reply:
Ok great!